### PR TITLE
Merge changes from Excel-Custom-Functions (TypeScript)

### PR DESCRIPTION
**Pull request by akrantz**

### CODE REVIEW

File: manifest.xml [hunk 1]
```diff
@@ -22,11 +22,6 @@
     <SourceLocation DefaultValue="https://localhost:3000/taskpane.html"/>
   </DefaultSettings>
   <Permissions>ReadWriteDocument</Permissions>
-  <Requirements>
-    <Sets DefaultMinVersion="1.1">
-        <Set Name="CustomFunctionsRuntime" MinVersion="1.1"/>
```
Comment (jayrathi) [inline manifest.xml]: Why is the requirement set not needed anymore?

Comment (akrantz) [reply manifest.xml]: Is was removed because of bug 3220358. Manifest validation fails with it included. Once that is fixed, we can put it back.

Comment (akrantz) [reply manifest.xml]: It's removed from the TypeScript template already. This just merges the same changes over tot he JavaScript template.

**Comment by TCourtneyOwen** [comment]
Just saw this PR.  Sorry I missed it

**Pull request merged**